src/WorkflowManager/Common/Interfaces/IPayloadService.cs

```diff
@@ -41,6 +41,12 @@ public interface IPayloadService : IPaginatedApi<Payload>
                                          string? patientId = "",
                                          string? patientName = "");
 
+        /// <summary>
+        /// Deletes a payload by id.
+        /// </summary>
+        /// <param name="payloadId">payload id to delete.</param>
+        Task<bool> DeletePayloadFromMinIOAsync(string payloadId);
```

Comment:
Please could you remove 'FromMinIO'. The storage layer is theoretically pluggable, so might not be MinIO.